calyx/src/ir/structure.rs

```diff
@@ -123,7 +123,7 @@ pub enum CellType {
 #[derive(Debug)]
 pub struct Cell {
     /// Name of this cell.
-    pub name: Id,
+    pub(super) name: Id,
```

Comment:
Does this need to be `pub(super)` or can we just make it private?

Reply:
Correction, should probably leave it as `pub(super)` since the builder creates cells and groups directly, unless we want to create explicit constructors for cell and group.